src/js/rollup.config.es5.js

```diff
@@ -36,7 +36,7 @@ export default {
           {
             modules: false,
             targets: {
-              browsers: '> 1%'
+              browsers: '> 0.5%'
```

Comment:
IE now has less than 1% market share so we have to bump this down to get critical automatic polyfilling